cmd/cl-adm/cmd/cmd.go

```diff
@@ -17,6 +17,7 @@ import (
 	"github.com/spf13/cobra"
 
 	"github.com/clusterlink-net/clusterlink/cmd/cl-adm/cmd/create"
+	initcmd "github.com/clusterlink-net/clusterlink/cmd/cl-adm/cmd/init"
```

Comment:
I think `initcmd` is redundant in this context.

Reply:
I can't import init- reserved word for function